beaker-rspec.gemspec

```diff
@@ -36,9 +31,7 @@ Gem::Specification.new do |s|
   # Dependency for example spec/acceptance tests
   s.add_runtime_dependency 'beaker-pe'
 
-  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.2.2')
-    s.add_development_dependency 'rack', '~> 1.6'
-  end
+  s.add_development_dependency 'rack', '>= 2.1.4'
```

Comment:
Why is this still needed? Previously it was pinned but this looks like it can simply be removed
```suggestion
```

Reply:
beaker had this feature to render documentation as html and disaplay it locally, rack might have been used for a local webserver.

Reply:
It looked like it was pinned on older Rubies.